packages/perseus/src/widgets/explanation.tsx

```diff
@@ -110,53 +111,15 @@ class Explanation extends React.Component<Props, State> {
                 </div>
             );
         } else {
-            if (isArticle) {
-                // NOTE: For articles, the baseElements `Link` may is either be
-                // an anchor tag (if rendered in a modal) or a Wonder Blocks
-                // link component in lesson page, see: `article-page.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.articleLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
-                            // HACK(michaelpolyak): WB Link doesn't passthrough
-                            // class name.
-                            style={styles.explanationLink}
-                            href={href}
-                            onClick={onClick}
-                            role="button"
-                            aria-expanded={this.state.expanded}
-                        >
-                            {`[${linkAnchor}]`}
-                        </Link>
-                    </div>
-                );
-            } else {
-                // NOTE: For exercises, the baseElements `Link` is an
-                // anchor tag, see: `perseus-api.jsx`.
-                linkContainer = (
-                    <div
-                        className={css(
-                            styles.linkContainer,
-                            styles.exerciseLink,
-                        )}
-                    >
-                        <Link
-                            className={css(styles.explanationLink)}
-                            href={href}
-                            onClick={onClick}
-                            role="button"
-                            aria-expanded={this.state.expanded}
```

Comment:
Do we not need `aria-expanded` anymore?

Reply:
I'll be doing another PR that completes the partial accessibility in this widget. Even though this attribute doesn't aid in accessibility without other related attributes, I've restored it in the `<Clickable>` element so that it's not forgotten and can be leveraged in the forthcoming PR.